collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -526,6 +536,70 @@ rke2-k8s() {
 
 }
 
+kubeadm-k8s() {
+  
+  KUBEADM_DIR="/etc/kubernetes/"
+  KUBEADM_STATIC_DIR="/etc/kubernetes/manifests/"
+  techo "Collecting k8s kubeadm cluster logs"
+  if [ -f /$USER/.kube/config ]; then
```

Comment:
It's possible that a user doesn't store the kubeconfig in this location. Can we make this configurable?

```
KUBECONFIG=${KUBECONFIG:-"$USER/.kube/config"}
```